### DISCUSSION TITLE
How to cite Photon?

### DISCUSSION THREAD

**Discussion by waeiski** [komoot/photon#829] [General]
Hey,

I am working on a research data and a subsequent academic article where I use Photon geocoding service through the geopy library, and was wondering what is the preferred/correct way to cite Photon? In my work, my current citation is just taken directly from GitHub as follows:

```
@misc{Komoot2024,
  title = {Komoot/Photon},
  author = {{Komoot}},
  year = {2024},
  url = {https://github.com/komoot/photon}
  address = {Potsdam, Germany},
  copyright = {Apache-2.0},
  howpublished = {komoot},
}
```

And as an APA citation, it looks like this: 
Komoot. (2024). Komoot/photon (Version 0.5.0) [Java]. komoot. https://github.com/komoot/photon (Original work published 2013)

But I would like to know what is the preferred way to cite the software, so if any of the developers can help, I would appreciate it.

Thank you!

**Comment by lonvia** [comment]
@christophlingg any preferences here?
